doc/proposals/openshift-4.3/operator-testing-tool.md

```diff
@@ -113,7 +116,24 @@ Tasks:
  * Update the CHANGELOG and Migration files with breaking changes and removals
 
 
+#### Story 5 - Support Custom Tests
+Tasks:
```

Comment:
There was more discussion about this today in the Group F meeting, and there's still not a consensus on how the tests should be designed, packaged, and delivered.

So task 1 here is to design the test format and interface. The external plugin design we have now may not be ideal for the pipeline use case since it:
* Depends on access to the operator's source
* Does not provide a robust solution for packaging test dependencies. 
* Does not provide isolation between the pipeline environment and the test execution environment.

One option that was discussed was for an operator developer to build an e2e image alongside (or as part of) their operator image such that tests could be executed in-cluster as Kubernetes `Job` workloads.

Reply:
done, I added a new story that covers the design aspects of the custom test feature.